resources/design-document.md

```diff
@@ -242,3 +265,16 @@ trainingSeriesName // partition key, string
 
 
 # 8. Pages
+![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide1.PNG)
```

Comment:
You'll want all of these to be relative references ... e.g. for this one `![](resources/Images/Slide1.PNG)`.

It'd also be helpful to add a heading/description for each one. E.g. "Welcome/Login Screen".